mne/chpi.py

```diff
@@ -411,9 +416,10 @@ def _calculate_chpi_positions(raw, t_step_min=0.1, t_step_max=10.,
         time_sl = slice(max(time_sl, 0),
                         min(time_sl + hpi['n_window'], len(raw.times)))
         with use_log_level(False):
-            meg_chpi_data = raw[hpi['picks'], time_sl][0]
-        this_data = meg_chpi_data[:-1]
-        chpi_data = meg_chpi_data[-1]
+            meg_chpi_data = raw[hpi['picks'], time_sl][0] # loads good channels with hpi_stim
+        mchpi_data = np.mean(meg_chpi_data[:-1],axis=1,keepdims=True)     
```

Comment:
unfortunately `keepdims` is not available on older `numpy` so `np.mean(..., 1)[:, np.newaxis]` should be used